extension/src/test/util/experiments.ts

```diff
@@ -0,0 +1,147 @@
+import {
+  Executor,
+  ExecutorState,
+  ExpData,
+  EXPERIMENT_WORKSPACE_ID,
+  ExperimentStatus,
+  ExpRange,
+  ExpShowOutput,
+  ExpState
+} from '../../cli/dvc/contract'
+
+export const generateTestExpData = (data: Partial<ExpData> = {}): ExpData => ({
```

Comment:
Should there be a couple tests for some of these utils? Might make debugging easier in the future 🤔